### PR TITLE
feat: use fr-container

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Close https://github.com/etalab/data.gouv.fr/issues/823

### CODE REVIEW

File: udata_front/theme/gouvfr/templates/reuse/display.html [hunk 1]
```diff
@@ -98,7 +98,7 @@ <h1 class="fr-mb-0 fr-h1">
         </div>
     </section>
 
-    <section class="fr-grid-row fr-grid-row--gutters fr-mt-4w fr-mb-6w container">
+    <section class="fr-grid-row fr-grid-row--gutters fr-mt-4w fr-mb-6w fr-container">
```
Comment (maudetes) [inline udata_front/theme/gouvfr/templates/reuse/display.html:101]: Woops, I have an issue with the reuse tabs container here. See:
image

**Review by maudetes — COMMENTED** [review]
Some minor comments :)

File: theme/less/content/callouts.less [hunk 2]
```diff
@@ -8,7 +8,7 @@ Callouts are areas that are meant to pop "out" of the page, usually with a colou
 
 ```callouts.html
```
[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply theme/less/content/callouts.less]: There was 2 callouts, one in content and the other in component. I removed this one.

File: udata_front/theme/gouvfr/templates/carousel.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {# First row #}
```
[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/templates/carousel.html:1]: De we want to remove the unused code ?

File: udata_front/theme/gouvfr/templates/reuse/display.html (hunk 1 above)
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/templates/reuse/display.html]: Done ! This was the only one with a different markup

File: udata_front/theme/gouvfr/templates/carousel.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (maudetes) [reply udata_front/theme/gouvfr/templates/carousel.html:1]: I think we don't want to bring carousel back. Right @agarrone?

**Review by maudetes — APPROVED** [review]
Looks good to me! :clap:

**Pull request merged**